### PR TITLE
Strip out commas in distribution limit to avoid crashing big number up stack

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Strip out commas in distribu tion limit to avoid crashing big number up stack.

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1804** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1804" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1804?utm_source=stack-comment).

File: src/components/v2/shared/DistributionSplitsSection/SpecificLimitModal.tsx [hunk 1]
```diff
@@ -25,7 +25,11 @@ export default function SpecificLimitModal({
   function setNewSplitsFromLimit() {
     form.validateFields()
 
-    setDistributionLimit(form.getFieldValue('distributionLimit'))
+    const distributionLimit = form
```
Comment (tomquirk) [inline src/components/v2/shared/DistributionSplitsSection/SpecificLimitModal.tsx:28]: @wraeth-eth i think we should fix this at a lower level. Ideally the `distributionLimit` value never contains commas. I think this is how other things work throughout the app? (no 100% sure)

Comment (wraeth-eth) [reply src/components/v2/shared/DistributionSplitsSection/SpecificLimitModal.tsx:28]: definitely agree - I have some stuff in the pipeline actually to fix this (I hope). I think we should still get this one out to prod asap, wdyt?

FWIW - currently reworking the form stuff to be more in line with what antd likes

Comment (tomquirk) [reply src/components/v2/shared/DistributionSplitsSection/SpecificLimitModal.tsx:28]: yeah k

**Review by tomquirk — APPROVED** [review]

**Pull request merged**